### ISSUE TITLE
iFrame title attributes 

### ISSUE THREAD

**Issue by ryanmccombe** [bbc/simorgh#3961]
**Describe the bug**
We have two iframes - the outer iframe and the inner SMP iframe, we need to add title attributes to both these iframes to match the screen reader UX - This should be the same whether it's SMP on a live radio page, MAP or article page.

**Expected behaviour**
Outer iframe - title attribute 'Video player' if it's video content or 'Audio player' if it's audio content. If we cannot tell what the media type is, it should be 'Media player'. The title text should be in the local language.

Inner SMP iframe - title attribute '[headline]'

Note there should not be a page title element contained within either of the iframes. - covered in https://github.com/bbc/simorgh/issues/4231

We should carry out a developer accessibility swarm with all [supported AT](https://bbc.github.io/accessibility-news-and-you/accessibility-and-supported-assistive-technology), note that iframes are not acknowledges with all screen readers.

**Comment by aeroplanejane** [comment]
Have requested translations for "audio player", "video player" and "media player" - we do not have a timescale for when these will be available - we should always use english as the default if no translation has been provided.

**Comment by pjlee11** [comment]
As part of this fix we can remove the `alpha` tag from the `psammead-media-player`
